simtools/camera_efficiency.py

```diff
@@ -541,7 +588,8 @@ def plot_cherenkov_efficiency(self):
 
         Returns
         -------
-        plt
+        fig
+            The figure
```

Comment:
I see the difficult of writing a good sentence here, but maybe we can discuss more what actually is returned? A matplotlob figure object? A pdf? An ascii figure? A path to a figure?

Reply:
That is right. I now looked at the return statement of the respective function and it is a pyplot.fig, so I suggest I write there "Instance of pyplot.figure", "pyplot.figure" or "instance of Figure" (then pyplot is omitted) which one would be better?

Reply:
`Instance of pyplot.figure` is best